test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/CommandRunnerTest.cs

```diff
@@ -24,7 +34,7 @@ public async Task Run_AlwaysCaptureStdout()
                 {
                     return Task.Run(() =>
                     {
-                        for (int i = 0; i < 1000; i++)
+                        for (int i = 0; i < 100; i++)
```

Comment:
This test was taking a minute to run so I reduced this value to get it to finish in a few seconds.  It still catches problems and is way faster.